autobahn/wamp/uri.py

```diff
@@ -282,29 +283,35 @@ def is_exception(self):
 
 
 @public
-def register(uri):
+def register(uri, **options):
     """
```

Comment:
Yes please! It is slightly more "verbose" or more "explicit" depending on how you look at it.